### PR TITLE
fabric: add page

### PR DESCRIPTION
**Pull request by mjherich**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** v1.4.107

### CODE REVIEW

**Comment by mjherich** [comment]
@danielmiessler Let me know what you think about this tldr

File: pages/common/fabric.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# fabric
+
+> An open-source framework for augmenting humans using AI.
+> Provides a modular framework for solving specific problems using a crowdsourced set of AI prompts.
+> More information: <https://github.com/danielmiessler/fabric>.
+
+- Run setup to configure fabric:
```
Comment (gutjuri) [inline pages/common/fabric.md:7]:
```suggestion
- Run the setup to configure fabric:
```

File: pages/common/fabric.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Run a custom user-defined pattern:
+
+`fabric --pattern {{custom_pattern_name}}`
+
+- Run a pattern and save the output to a file:
+
+`fabric --pattern {{pattern_name}} --output {{path/to/output_file}}`
+
+- Run a pattern with specified variables:
```
Comment (gutjuri) [inline pages/common/fabric.md:35]:
```suggestion
- Run a pattern with the specified variables:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks, I have some suggestions.

**Review by nelsonfigueroa — APPROVED** [review]
Hello, thanks for you contribution! While there are some suggestions at the moment, I personally think this PR is good. I will leave it up to others to decide if the suggestions are necessary.

**Review by spageektti — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**